test/Microsoft.AspNet.Mvc.FunctionalTests/compiler/resources/HtmlGenerationWebSite.HtmlGeneration_Home.EmployeeList.html

```diff
@@ -94,5 +94,5 @@
 <option>1002</option>
 </select>
 </div>            <input type="submit" />
-</form></body>
+<input name="[0].Remote" type="hidden" value="false" /><input name="[1].Remote" type="hidden" value="false" /><input name="[2].Remote" type="hidden" value="false" /></form></body>
```

Comment:
Ah you're talking specifically about revisiting our decision to write these additional `<input/>` (or `<input>`) elements identically to the check boxes. Right?